src/Surfnet/StepupMiddleware/ApiBundle/Authorization/Service/InstitutionAuthorizationService.php

```diff
@@ -69,12 +64,36 @@ public function __construct(
      *
      * The additional test is performed to indicate if the actor is SRAA.
      *
-     * @param Institution $actorInstitution
+     * @param IdentityId $actorId
      * @param InstitutionRoleSet $roleRequirements
+     * @return InstitutionAuthorizationContext
+     */
+    public function buildInstitutionAuthorizationContext(IdentityId $actorId, InstitutionRoleSet $roleRequirements)
```

Comment:
In the doc block of this method it states:
> The additional test is performed to indicate if the actor is SRAA.

There are no tests of that sort in this method. Copy pasta?

Reply:
The check is still there on line 79:80.